### PR TITLE
Bennett

### PR DESCRIPTION
**Pull request by Benobbi**
added the file for task 1, wanted to your thoughts on them before we merge with the main branch

### CODE REVIEW

File: main.h [hunk 1]
```diff
@@ -9,5 +9,8 @@ int _printf(const char *format, ...);
 /* HELPER FUNCTIONS */
 int print_char(char c);
 int print_string(char *str);
+int _putchar(char c);
+int print_char(va_list args);
+{
```
Comment (angelotheman) [inline main.h]: There's this ending `{`

File: main files/0-main.c
Comment (angelotheman) [inline main files/0-main.c:1]: We can't push this `main.c` file. It's only for testing

File: main files/2-main.c
Comment (angelotheman) [inline main files/2-main.c:1]: We can't push this as it's for testing only

**Review by angelotheman — CHANGES_REQUESTED** [review]
I've commented on those that needs the fixing. Kindly resolve and let's move on.

Cheers to 🍕

**Comment by angelotheman** [comment]
Less I forget, kindly change the name of your helper functions so that it doesn't conflict

**Review by Benobbi — COMMENTED** [review]
keeping the main.c files in a subfolder keeps them from being checked. the checks are done in the root of the repo.

File: main.h (hunk 1 above)
Comment (Benobbi) [reply main.h]: removed the bracket at the end

**Review by Benobbi — COMMENTED** [review]
deleted the curly bracket at the end of the main.h file. for the main.c files it should be fine as long as they are not in the root of the repo, we can also delete them altogether.

**Review by angelotheman — APPROVED** [review]
I'm approving these but I would edit in the merge conflicts. Some of them are just test cases. We would separate them into folders of their own

**Pull request merged**